data-raw/tranform_inputs_complete.R

```diff
@@ -99,9 +99,26 @@ flow_cfs_2019_biop_itp <- flow_cfs_2019_biop_itp |>
   left_join(moke_2019) |> # add in Moke
   select(date:`Cosumnes River`, `Mokelumne River`, `Merced River`:`San Joaquin River`) # reorder
 
-# create flow_cfs with both 2008-2009 biop and 2018-2019 biop/itp ---------------
+# Add run of river
+calsim_run_of_river <- read_rds('data-raw/calsim_run_of_river/run_of_river_r2r_calsim.rds')
+
+# TODO: temporarily just use 2019 data for Moke in run of river
+moke_2019 <- read_excel('data-raw/calsim_2019_BiOp_ITP/EBMUDSIM/CVPIA_SIT_Data_RequestUpdated2022.xlsx',
```

Comment:
I am also getting a new names note here. Its probably fine because we do not use that node nut would be good to double check.